app/src/org/koreader/launcher/device/lights/TolinoWarmthController.kt

```diff
@@ -74,15 +76,24 @@ class TolinoWarmthController : LightInterface {
             Logger.w(TAG, "warmth value of of range: $warmth")
             return
         }
-        val colorFile = File(COLOR_FILE)
-        Logger.v(TAG, "Setting warmth to $warmth")
-        try {
-            colorFile.setLastModified(System.currentTimeMillis())
-        } catch (e: Exception) {
-            Runtime.getRuntime().exec("su -c chmod 666 $COLOR_FILE")
+
+        // on the first call, we check on which Tolino we are running
+        if (COLOR_FILE=="") { 
```

Comment:
Codacy had a little complaint:
```suggestion
        if (COLOR_FILE == "") {
```

(also spaces around `=` on later lines)

Seems reasonable to me, pinging @pazos